single.php

```diff
@@ -1,12 +1,24 @@
 <?php
 get_header();
-get_template_part('templates/header-quote', 'tpl');
+
+$blog = new CMB2Fields(get_the_ID());
+$render_args = [
+  'title' => get_the_title(),
```

Comment:
You don't need to set these as render args so can pretty much just delete lines 5 - 8